AI/StupidAI/StupidAI.cpp

```diff
@@ -49,7 +49,8 @@ struct EnemyInfo
 	const CStack * s;
 	int adi, adr;
 	std::vector<BattleHex> attackFrom; //for melee fight
-	EnemyInfo(const CStack * _s) : s(_s), adi(0), adr(0)
+	EnemyInfo(const CStack * _s)
+		: s(_s), adi(0), adr(0)

```

Comment:
It's not that simple, as some constructors have dozens of parameters and take up a lot of vertical space.